core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -2161,9 +2161,12 @@ private static RelDataType deriveTypeMapFromEntries(SqlOperatorBinding opBinding
   public static final SqlFunction POW =
       SqlStdOperatorTable.POWER.withName("POW");
 
+  /** The "TRUNC(numeric1 [, integer2])" function. Identical to the standard <code>TRUNCATE</code>
+  * function except the return type should be a double if numeric1 is an integer. */
   @LibraryOperator(libraries = {BIG_QUERY})
-  public static final SqlFunction TRUNC =
-      SqlStdOperatorTable.TRUNCATE.withName("TRUNC");
+  public static final SqlFunction TRUNC = SqlStdOperatorTable.TRUNCATE
```

Comment:
Consider renaming the function to TRUNC_BIG_QUERY. The BigQuery FLOOR and TRUNC functions are now in sync, but need to remain so, and linking their names will make that more likely. 

In Jira please link to CALCITE-5747. That's where these came adrift in the first place.

Reply:
I think renaming it might be a bit redundant, I only renamed FLOOR and CEIL because they have standard counterparts with the same name, TRUNC however is unique from the standard TRUNCATE. What do you think? I don't feel very strongly either way. @julianhyde

Reply:
And I think the JIRA should already have a link to 5747 let me know if it looks wrong

Reply:
maintainers will do the wrong thing if you don't make it very obvious. Find a way to make it obvious.